socorro/external/boto/crashstorage.py

```diff
@@ -221,15 +204,6 @@ class TelemetryBotoS3CrashStorage(BotoS3CrashStorage):
 
     """
 
-    required_config = Namespace()
-    required_config.add_option(
-        "resource_class",
-        default="socorro.external.boto.connection_context.RegionalS3ConnectionContext",
-        doc="fully qualified dotted Python classname to handle Boto connections",
-        from_string_converter=class_converter,
-        reference_value_from="resource.boto",
-    )
-
```

Comment:
This is redundant since the superclass already sets the default for `resource_class` as `RegionalS3ConnectionContext`.